heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesController.java

```diff
@@ -43,114 +49,110 @@ public KubernetesController(
    *
    * @return success
    */
-  public boolean killTopology() {
+  protected boolean killTopology() {
```

Comment:
Yeah, since I'll need to re-do the tests here, I'd prefer to do a separate PR here.